packages/react/package.json

```diff
@@ -42,6 +42,6 @@
     "@remote-ui/rpc": "^1.3.0",
     "@types/react": ">=17.0.0 <18.0.0",
     "@types/react-reconciler": "^0.26.0",
-    "react-reconciler": ">=0.26.0 <1.0.0"
+    "react-reconciler": ">=0.26.0 <=0.26.2"
```

Comment:
Could we make this `<0.27.0` instead? I doubt it will make a difference in practice, but seems to be more what we are after here.